etc/hub.api.md

```diff
@@ -32,9 +34,13 @@ import { SetupMailboxResponse } from '@textile/users-grpc/api/usersd/pb/usersd_p
 import { WriteTransactionReply } from '@textile/threads-client-grpc/threads_pb';
 import { WriteTransactionRequest } from '@textile/threads-client-grpc/threads_pb';
 
+export { AbortController_2 as AbortController }
+

```

Comment:
Strange these changes weren't already in, I guess from a previous PR. Maybe someone forgot to run `npm run docs`.